string scene=MCWorld
int64 timestamp
mob[] mobs
